src/bindings/python/src/openvino/runtime/ie_api.py

```diff
@@ -265,12 +198,71 @@ def infer_new_request(self, inputs: Union[dict, list, tuple, Tensor, np.ndarray]
         # overloaded functions of InferRequest class
         return self.create_infer_request().infer(inputs)
 
-    def __call__(self, inputs: Optional[Union[dict, list]] = None) -> dict:
+    def __call__(self,
+                 inputs: Union[dict, list, tuple, Tensor, np.ndarray] = None,
+                 shared_memory: bool = True) -> dict:
         """Callable infer wrapper for CompiledModel.
 
-        Take a look at `infer_new_request` for reference.
+        Infers specified input(s) in synchronous mode.
+
+        Blocks all methods of CompiledModel while request is running.
+
+        Method creates new temporary InferRequest and run inference on it.
+        It is advised to use a dedicated InferRequest class for performance,
+        optimizing workflows, and creating advanced pipelines.
+
+        This method stores created `InferRequest` inside `CompiledModel` object,
+        which can be later reused in consecutive calls.
+
+        The allowed types of keys in the `inputs` dictionary are:
+
+        (1) `int`
+        (2) `str`
+        (3) `openvino.runtime.ConstOutput`
+
+        The allowed types of values in the `inputs` are:
+
+        (1) `numpy.array`
```

Comment:
`numpy.array` should be replaced by `numpy.ndarray`. It was an opening to me that some other types can also work, e.g. `torch.Tensor` which is cast to numpy.ndarray implicitly. Based on that I would rephrase the item `(1)` to:
```
(1) `numpy.ndarray` and all the types that are castable to it, e.g. `torch.Tensor`.
```

Reply:
Done!